Astral Pathfinder/Game/UIManager.hpp

```diff
@@ -37,17 +37,24 @@ class UIManager {
   // MARK: - UIManager Fields
   // TODO: Implement main menu and endscreen
   // MainMenu mainMenu;
-  PlanetInfo selectedPlanetInfo, currentPlanetInfo;
+  PlanetInfo selectedPlanetInfo, DockedPlanetInfo;
   ShipInfo shipInfo;
-  int window = -1;
-  bool oneClean = true, twoClean = true;
+  enum {
+    none, currentPlanetWindow, selectedPlanetWindow
+  } currentWindow = none;
+  
+  enum {
```

Comment:
This can be declared inside of the handleMouseDown() method since that's the only m it is used.